stability/cassandra_stress.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -o pipefail
+
+# General env
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/../metrics/lib/common.bash"
+
+IMAGE="docker.io/library/cassandra:latest"
+CONTAINER_NAME="${CONTAINER_NAME:-cassandra_test}"
+DOCKER_IMAGE="cassandra:latest"
+PAYLOAD_ARGS="${PAYLOAD_ARGS:-tail -f /dev/null}"
+
+function main() {
+	local cmds=("docker")
+
+	init_env
+	check_cmds "${cmds[@]}"
+	sudo -E docker pull "${DOCKER_IMAGE}"
+	sudo -E docker save -o "${DOCKER_IMAGE}.tar" "${DOCKER_IMAGE}"
+	sudo -E ctr i import "${DOCKER_IMAGE}.tar"
+
+	sudo -E ctr run -d --runtime "${CTR_RUNTIME}" "${IMAGE}" "${CONTAINER_NAME}" sh -c "${PAYLOAD_ARGS}"
+	CMD="cassandra -R"
```

Comment:
Hi @GabyCT could it be better to declare CMD constant as globa i.e. outside `main()`? in order to have more visibility.

Reply:
@dborquez thanks for the feedback changes applied